boa/src/vm/instructions.rs

```diff
@@ -100,6 +105,10 @@ impl std::fmt::Display for Instruction {
             Self::Neg => write!(f, "Neg"),
             Self::BitNot => write!(f, "BitNot"),
             Self::Not => write!(f, "Not"),
+            Self::DefVar(name) => write!(f, "DefVar({})", name),
+            Self::DefLet(name) => write!(f, "DefLet({})", name),
+            Self::DefConst(name) => write!(f, "DefConst({})", name),
+            Self::InitLexical(usize) => write!(f, "InitLexical({})", usize),
```

Comment:
I would prefer if the binding wasn't called `usize`.

Reply:
Yeah I agree.